modules/custom-css/custom-css-in-customizer.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+class Jetpack_Custom_CSS_Customizer {
+	static $theme_mod_update_counter = 0;
+	static $update_theme_mods = array();
+	static $setting_order = array();
+
+	public static function init() {
+	//	if ( ! CustomDesign::is_active_or_preview() ) {
+	//		return;
+	//	}
+
+		if ( ! apply_filters( 'enable_custom_customizer', true ) ) {
+			return;
+		}
```

Comment:
Should consolidate this filter and `CustomDesign::is_active_or_preview()` as hooks onto a new filter like `jetpack_enable_css_in_customizer`